testutil/apptesting/test_suite.go

```diff
@@ -37,12 +38,10 @@ type KeeperTestHelper struct {
 // Setup sets up basic environment for suite (App, Ctx, and test accounts)
 func (s *KeeperTestHelper) Setup() {
 	s.App = testutil.Setup(s.T()).(*app.App)
-	ctx := s.App.GetBaseApp().NewContext(
-		false,
-	)
+	ctx := s.App.GetBaseApp().NewUncachedContext(false, tmtypes.Header{Height: 1, ChainID: "neutron-1", Time: time.Now().UTC()})
```

Comment:
Could you add a small comment here that describes why do we need such complex context?